### DISCUSSION TITLE
訓練員過世！警犬去墓園「突有蝴蝶停在鼻子上」，牠反應讓網淚崩

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1765] [General]
狗狗的護衛心通常很重，如果有陌生訪客進門，狗狗都是第一個發現並大聲汪汪叫示警，提醒主人有外人來了。偶爾也會出現狗狗莫名對著窗外空氣吠叫的情況，有些主人會認為是動物感知「無形」的能力比人類靈敏。微博上流傳一段警犬的感人影片，牠的訓練員過世了，而狗狗去墓園的反應令人心疼。

根據中媒《大眾網東營》報導，有隻擔任警犬的德國牧羊犬被帶到墓園裡，原來其他人是想去向已故的訓犬員致意，把他生前訓練過的警犬也一併帶去。通常狗狗到戶外會很興奮，看到什麼小動物都立刻狂奔起來，但是有蝴蝶迎面飛來，警犬卻一點也不躁動。

色彩鮮艷的蝴蝶輕輕停在狗狗的鼻子上，警犬卻絲毫不緊張，沒有甩甩頭或出手趕走蝴蝶，反而是安靜地坐下，瞇著眼睛露出平靜療癒的表情，好像牠和這隻蝴蝶已經認識了好久好久。或許是警犬性格特別穩定，也許是萬物皆有靈，這一幕帶給圍觀者不同的解讀和意義。

![Uploading 警犬5-5.jpg…]()

資料來源：https://www.teepr.com/1721135/valentinawu/警犬與蝴蝶/

看完這篇新聞後，覺得動物都是很有靈性，他們這一輩子就只有你，所以要好好對待他們
可以不喜歡動物，但是不要傷害他們！

**Comment by mrweixd** [comment]
真的很想養一隻聰明的狗，用語言以外的東西產生感情與聯繫實在是非常美麗的故事。
